DEPS

```diff
@@ -13,7 +13,7 @@ deps = {
   "vendor/bip39wally-core-native": "https://github.com/brave-intl/bip39wally-core-native.git@13bb40a215248cfbdd87d0a6b425c8397402e9e6",
   "vendor/bat-native-anonize": "https://github.com/brave-intl/bat-native-anonize.git@e3742ba3e8942eea9e4755d91532491871bd3116",
   "vendor/bat-native-tweetnacl": "https://github.com/brave-intl/bat-native-tweetnacl.git@800f9d40b7409239ff192e0be634764e747c7a75",
-  "vendor/bat-native-usermodel": "https://github.com/brave-intl/bat-native-usermodel.git@865ba342737c09b13ee18e45b8ece5970bb77246",
+  "vendor/bat-native-usermodel": "https://github.com/brave-intl/bat-native-usermodel.git@02b8c81c94072c67fe00108feb90786e088d4d26",
```

Comment:
?

Reply:
Deps were fixed in `bat-native-usermodel`: https://github.com/brave-intl/bat-native-usermodel/issues/31 - I'll move this to a separate commit.

Reply:
we should move this into brave-core, or at the very least move the gn build config into brave-core

Reply:
cc: @tmancey @dimmu 

Filed an issue: https://github.com/brave/brave-browser/issues/10794